Incorrect 2021 AMT amount

Issue #1225 has already pointed out that the US AMT parameters have not been specified for the years after 2019.  This test case illustrates the consequences of this problem: the case is simulated to have a positive AMT in 2021 ($4054) when neither TAXSIM35 nor USI-Tax-Analyzer generate a positive AMT amount and those two models generate the same US income tax liability of $85,538.

Here is the YAML test file, below which are the test results:
```
# OpenFisca-US test file derived from r21.ita.csv and r21.ota.csv files
- name: Tax unit with recid 12233 from r21.ita.csv
  absolute_error_margin: 0.01
  period: 2021
  input:
    people:
      person1:
        is_tax_unit_head: 1
        is_tax_unit_spouse: 0
        is_tax_unit_dependent: 0
        age: 55
        ssi: 0
        state_supplement: 0
        wic: 0
        employment_income: 177000
        long_term_capital_gains: 100000
      person2:
        is_tax_unit_head: 0
        is_tax_unit_spouse: 1
        is_tax_unit_dependent: 0
        age: 58
        ssi: 0
        state_supplement: 0
        wic: 0
        employment_income: 176000
    spm_units:
      spm_unit:
        members: [person1, person2]
        snap: 0
        tanf: 0
    tax_units:
      tax_unit:
        members: [person1, person2]
        premium_tax_credit: 0
    households:
      household:
        members: [person1, person2]
        state_code: TX
  output:
    taxsim_tfica: 23752.70
    income_tax: 85538.00
```

The above YAML test file generates the following results:
```
================================= FAILURES ===================================
_______________________________ test session _________________________________
/Users/mrh/work/r21-id12233.test.yaml:
  Test 'Tax unit with recid 12233 from r21.ita.csv':
    income_tax@2021: [97468.] differs from 85538.0 
      with an absolute margin [11930.] > 0.01
========================= short test summary info ============================
FAILED r21-id12233.test.yaml::
============================== 1 failed in 1.19s ===============================
```

--- a/changelog_entry.yaml
+++ b/changelog_entry.yaml
@@ -1,0 +1,4 @@
+- bump: patch
+  changes:
+    added:
+      - Regression test for AMT calculation against TAXSIM35 (issue #1279).

--- a/policyengine_us/tests/policy/baseline/gov/irs/tax/federal_income/alternative_minimum_tax/integration.yaml
+++ b/policyengine_us/tests/policy/baseline/gov/irs/tax/federal_income/alternative_minimum_tax/integration.yaml
@@ -1,0 +1,26 @@
+# Integration test from GitHub issue #1279
+# Validates AMT calculation against TAXSIM35
+- name: Issue 1279 - Joint filers with employment income and capital gains
+  period: 2021
+  absolute_error_margin: 1
+  input:
+    people:
+      person1:
+        age: 55
+        employment_income: 177_000
+        long_term_capital_gains: 100_000
+      person2:
+        age: 58
+        employment_income: 176_000
+    tax_units:
+      tax_unit:
+        members: [person1, person2]
+    households:
+      household:
+        members: [person1, person2]
+        state_code: TX
+  output:
+    # TAXSIM35 and USI-Tax-Analyzer both produce zero AMT for this case
+    alternative_minimum_tax: 0
+    # Expected income tax from TAXSIM35
+    income_tax: 85_538

